packages/tsconfig-reference/copy/en/options/skipLibCheck.md

```diff
@@ -3,3 +3,12 @@ display: "Skip Lib Check"
 ---
 
 Skip type checking of declaration files.
+
+This can save time in a compilation at the expense of accuracy in your type system. For example, two libraries could
+define two copies of the same `type` in an inconsistent way. Rather than doing a full check of all d.ts files, TypeScript 
```

Comment:
Nah in this case I mean the literal `type` -

```ts
// in a global lib
type MyThing = { a: string }

// in user code
type MyThing = { b: string }
```

should raise an error but it won't - with `interface`s it's kinda fine because of how they expand each other

Reply:
Ah, so maybe replace `type` with `type foo` (since it looks like my misreading is an easy mistake to make)?